azure-pipelines.yml

```diff
@@ -8,7 +8,7 @@ pool:
 steps:
 - task: UsePythonVersion@0
   inputs:
-    versionSpec: 3.7.4
+    versionSpec: 3.7.5
```

Comment:
This is the second time builds started failing because of this exact issue.  As per my previous PR comment for MattR ;-)

https://docs.microsoft.com/en-us/azure/devops/pipelines/tasks/tool/use-python-version?view=azure-devops#arguments

Let's use a range, not a specific version.

Reply:
Thanks - didn't realize range was an option